src/components/displaySettings/displaySettings.template.html

```diff
@@ -131,6 +131,18 @@ <h2 class="sectionTitle">
         <select id="selectTheme" is="emby-select" label="${LabelTheme}"></select>
     </div>
 
+    <div class="checkboxContainer checkboxContainer-withDescription followSystemTheme">
+        <label>
+            <input type="checkbox" is="emby-checkbox" id="followSystemTheme" />
+            <span>Follow System Theme</span>
+        </label>
+        <div class="fieldDescription checkboxFieldDescription">Follow System Light/Dark Theme</div>
+    </div>
+
+    <div class="selectContainer">
```

Comment:
Same for this, hide it and show it only if reported as supported by the client.

Reply:
Will Do. Thanks